src/clang.rs

```diff
@@ -37,8 +37,16 @@ impl Cursor {
         };
 
         // Try to undo backend mangling
-        if cfg!(target_os = "macos") || cfg!(all(target_os = "windows", target_env = "gnu")) {
+        if cfg!(target_os = "macos") {
             mangling.remove(0);
+        } else if cfg!(target_os = "windows") {
+            // On windows, undo backend mangling only for gcc
```

Comment:
Just saw you added it in the caller, fine, but now it will remove two characters on mac, right?